index.html

```diff
@@ -811,25 +818,20 @@ <h3>The <dfn>toJSON</dfn> member</h3>
   <h3>The <dfn>PressureObserverOptions</dfn> dictionary</h3>
   <pre class="idl">
     dictionary PressureObserverOptions {
-      double sampleRate = 1.0;
+      [EnforceRange] unsigned long sampleInterval = 0;
     };
   </pre>
   <section>
-    <h3>The <dfn>sampleRate</dfn> member</h3>
+    <h3>The <dfn>sampleInterval</dfn> member</h3>
     <p>
-      The {{PressureObserverOptions/sampleRate}} member represents the <dfn>requested sampling
-      rate</dfn> expressed in Hz, ie. it represents the number of samples requested to be obtained
-      from the hardware per second. The [=reporting rate=] will never exceed the [=requested sampling rate=].
+      The {{PressureObserverOptions/sampleInterval}} member represents the [=requested sampling
+      interval=] expressed in milliseconds, ie. it represents the requested interval between samples
+      to be obtained from the hardware. The [=reporting rate=] will never exceed the sampling rate which
+      is obtained by dividing 1000 by the [=requested sampling interval=].
```

Comment:
```suggestion
      interval=] expressed in milliseconds. 
```

- I think the _"it represents the requested interval between samples to be obtained from the hardware"_ part should be in Concepts, as it's part of the explanation of this concept.
- I removed the second part because it is repeating something that is already said in Concepts.